### PR TITLE
min version of R is now 3.5

### PR DESCRIPTION
**Pull request by mattansb**
https://github.com/easystats/easystats/issues/240

### CODE REVIEW

**Review by IndrajeetPatil — CHANGES_REQUESTED** [review]
Make sure to also note this in `NEWS` as a breaking change.

File: tests/testthat/test-eta_squared.R [hunk 1]
```diff
@@ -615,6 +615,8 @@ if (require("testthat") && require("effectsize")) {
     expect_output(print(out), "Type II")
 
     skip_if_not_installed("car")
+    skip_if(getRversion() <= "3.6" &&
+              Sys.info()["sysname"] == "Darwin")
```
Comment (strengejacke) [inline tests/testthat/test-eta_squared.R:619]: you could also use `skip_on_os("mac")`?

Comment (mattansb) [reply tests/testthat/test-eta_squared.R:619]: But it does work on newer version of R on Mac 🤷‍♂️

File: NEWS.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # effectsize 0.7.0.9
 
+## Breaking Changes
+
+`effectsize` now requires minimal *`R`* version of `3.4`.
```
Comment (etiennebacher) [inline NEWS.md:5]: `3.5` instead, right?

Comment (mattansb) [reply NEWS.md]: hehe oops

File: tests/testthat/test-eta_squared.R (hunk 1 above)
Comment (strengejacke) [reply tests/testthat/test-eta_squared.R:621]: Yes, I just meant to replace sys.info

**Pull request merged**